src/containers/cellPairTraversals/SlicedTraversal.h

```diff
@@ -122,7 +119,7 @@ inline void SlicedTraversal<ParticleCell, CellFunctor>::traverseCellPairs() {
   // use fallback version if not applicable
   // FIXME: Remove this as soon as other traversals are available
   if (not isApplicable()) {
```

Comment:
should no longer be needed

Reply:
866924523f0f501f9c7dee2959365be3da234841